example/pubspec.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
   provider: ^6.0.1
   system_theme: ^1.0.1
   bitsdojo_window: ^0.1.1+1
+  window_manager: ^0.1.6
   flutter_acrylic: ^0.1.0
```

Comment:
When I test on Linux, upgrade to version 1.0.0+1 fix the problem.

Have you tried on Linux ?